app/controllers/qsets_controller.rb

```diff
@@ -52,8 +52,11 @@ def show
   def create
     create_group_params = params.permit(:name, :parent_id)
     @qset = Qset.new(create_group_params)
+    qset_type = params.permit(:qset_type)
+    # set qset type from new qset modal
+    @qset.settings(:permissions).qset_type = qset_type[:qset_type]
     @qset.save
-    redirect_to qset_path(@qset), notice: "Qset '#{@qset.name}' created."
+    redirect_to qset_path(@qset.id), notice: "Qset '#{@qset.name}' created."
```

Comment:
Yeah, it doesn't matter -- people prefer `@qset` in the rails word because it's more concise. See http://guides.rubyonrails.org/routing.html#generating-paths-and-urls-from-code.